### PR TITLE
Mixpanel stuff.

### PR DESCRIPTION
**Pull request by dkorolev**
Default Mixpanel token and tracking of `User` events as well.

`TODO(dkorolev): Unify those POST-s and JSON generation.`

@sompylasar : PTAL.

### CODE REVIEW

File: demo.cc [hunk 1]
```diff
@@ -632,50 +708,11 @@ class MixpanelUploader final {
...
-
-      // (reserved) The time of the event, in seconds.
-      uint64_t time;
-
-      // Question identifier.
-      schema::QID qid;
-
-      // Answer identifier.
-      schema::ANSWER answer;
+  inline void operator()(schema::UserRecord& u) { PushMixpanelEvent(MixpanelEvent::User(mixpanel_token_, u)); }
```
Comment (sompylasar) [inline demo.cc:711]: Weird formatting, this goes in one line, the next one in multiple lines. I assume this is what `make indent` did.

Comment (dkorolev) [inline demo.cc:711]: I noticed this too. Will get cleaned up very soon as I touch this part.

**Comment by sompylasar** [comment]
Mixpanel can store persistent info about users in addition to events. Do we want to use it to store the current state of the answers?

**Comment by dkorolev** [comment]
Yes, eventually.

As of today, I only want to be able to show that events are flying from TailProduce to Mixpanel.

**Comment by sompylasar** [comment]
@dkorolev Then LGTMerge.

**Comment by dkorolev** [comment]
Thank you!

Merging and sending you one small commit right now.

**Pull request merged**